_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Something old, something new…"
+date: 2018-06-20 12:00:00
+categories: preview
+tags: HackIllinois2018, HackIllinois2019, featured
+author: Kyle Begovich
+---
+
+Even though it's only June, planning for HackIllinois 2019 is already well under way, and our Summer Team would love to offer you an inside look at some of our motivations and guiding principles.
+
+# Something Old
```

Comment:
This section is largely positive. This section is a great opportunity to mention some of the biggest shortcomings of HackIllinois 2018 that we plan to address this go around.

Reply:
I would generally agree, but the entire post is very positive. I don't think the general audience has good enough memory to recall specific shortcomings unless they affected that individual directly, and I think harping on that before looking forward wouldn't be the play